doc/readme.eu.md

```diff
@@ -0,0 +1,135 @@
+# mupl - MangaDex Bulk Uploader
+Karpetak eta (.zip/.cbz) fitxategiak igotzen ditu MangaDexera modu azkar eta errazean.
+
+Jatorrizko bertsioa:
+[English](/readme.md)
```

Comment:
link back to English should be `../readme.md`

Reply:
Tested it before and it worked, but applied the suggested changes...
PS: The other languages point to the wrong location (doc/readme.md), want me to fix them?